integration-tests/src/test/scala/tech/beshu/ror/integration/suites/FilterRuleSuite.scala

```diff
@@ -83,8 +100,9 @@ object FilterRuleSuite {
 
   private def add3Docs(adminRestClient: RestClient, index: String, `type`: String): Unit = {
     val documentManager = new DocumentManagerJ(adminRestClient)
-    documentManager.insertDocAndWaitForRefresh(s"/$index/${`type`}/1", s"""{"db_name":"db_user1"}""")
-    documentManager.insertDocAndWaitForRefresh(s"/$index/${`type`}/2", s"""{"db_name":"db_user2"}""")
-    documentManager.insertDocAndWaitForRefresh(s"/$index/${`type`}/3", s"""{"db_name":"db_user3"}""")
+    documentManager.insertDocAndWaitForRefresh(s"/$index/${`type`}/1", s"""{"db_name":"db_user1", "code": 1}""")
```

Comment:
please use DocumentManager instead, so you can get rid of esVersion check in method above